packages/terra-popup/src/_PopupContent.jsx

```diff
@@ -191,14 +200,16 @@ class PopupContent extends React.Component {
       ...customProps
     } = this.props;
 
+    const { intl } = this.context;
```

Comment:
Move the intl and close declarations inside the block:
```
if (isFullScreen && !isHeaderDisabled) {
```

Reply:
Done!